test/e2e/storage/external/external.go

```diff
@@ -84,7 +84,16 @@ type driverDefinition struct {
 		// snapshotter class with DriverInfo.Name as provisioner.
 		FromName bool
 
-		// TODO (?): load from file
+		// FromFile is used only when FromName is false.  It
+		// loads a snapshot class from the given .yaml or .json
+		// file. File names are resolved by the
+		// framework.testfiles package, which typically means
+		// that they can be absolute or relative to the test
+		// suite's --repo-root parameter.
+		//
+		// This can be used when the snapshot class is meant to have
+		// additional parameters.
+		FromFile string
```

Comment:
Can we also add a "From Existing" option, similar to Storage Class above?

Reply:
Good suggestion. I will work on it.